CONTRIBUTING.md

```diff
@@ -127,6 +127,16 @@ JupyterLab v3.0.0
         jupyter-resource-usage v0.1.0 enabled OK
 ```
 
+There is an example config file included.
```

Comment:
I would worry that this file will diverge in the future if it is not covered by the tests. Because tratitlets can auto-generate th econfig I would suggest not including it.